<launch>
  <param name="halfWidth"     type="double"   value="0.25" />
  <param name="trackShape"  type="str"    value="oval_iri"/>
  
  <node pkg="plotter" type="MPC_track_plot.py" name="MPC_plotter" output="screen">
  <param name="loop_rate"     		 	 type="int"      value="1000" />
  <param name="track_visualization"   	 type="bool"      value="true" />
  <param name="sim"					     type="bool"      value="false" />
  <param name="vehicle_visualization"    type="bool"      value="true" />
  <param name="states_visualization"     type="bool"      value="false" />
  <param name="control_visualization"    type="bool"      value="true" />
  <param name="error_visualization"    type="bool"      value="false" />
  <param name="record_data"     		 type="bool"      value="false" />

  </node>

</launch>
